### PR TITLE
add postgres-dbaas to makefile for cluster testing

### PR DESCRIPTION
**Pull request by tobybellwood**
This doesn't require a version bump, as it is only a makefile change that affects the services brought up with a lagoon-remote install.

### CODE REVIEW

**Comment by tobybellwood** [comment]
ah - this will fail until I merge https://github.com/amazeeio/lagoon/pull/2311 into main to fix up the tests image (I think)

**Comment by tobybellwood** [comment]
@smlx - I've removed the tests from this PR, as they won't work until https://github.com/amazeeio/lagoon/pull/2311 is merged (which won't work until this PR is merged!)  Once this PR is merged here, I'll get the lagoon one passing, and then submit a new PR here for the tests.  phew...

File: charts/lagoon-remote/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ type: application
 # This is the chart version. This version number should be incremented each
 # time you make changes to the chart and its templates, including the app
 # version.
-version: 0.10.0
+version: 0.10.1
```
Comment (smlx) [inline charts/lagoon-remote/Chart.yaml:21]: This isn't required since there's no longer any change to the chart.

**Review by smlx — APPROVED** [review]
LGTM. Since this is now only touching the `Makefile` there isn't even a chart release.

I'd say just remove the version bump on `lagoon-remote` and merge!

**Comment by smlx** [comment]
@tobybellwood Here's how I've been doing these inter-dependent merges on Lagoon and the Charts.

1. Make the change required in the charts repo and push a branch.
2. Push a PR to Lagoon which also temporarily changes the `CHARTS_TREEISH` to your branch name from 1. Get this green, and merge it.
3. Create a PR in Charts repo, get it green and merge it.
4. Create PR back in Lagoon pointing at the tag from 3.

It's a bit complex but keeps each repo "locked" on a known-working version of the other.

File: charts/lagoon-remote/Chart.yaml (hunk 1 above)
Comment (tobybellwood) [reply charts/lagoon-remote/Chart.yaml]: It was needed when I had a test in there, but not anymore

**Comment by tobybellwood** [comment]
I'll test it using CHARTS_TREEISH anyway first :-)

**Review by smlx — APPROVED** [review]

**Pull request merged**